modules/cnf-cpu-infra-container.adoc

```diff
@@ -36,3 +74,5 @@ spec:
 <2> Specify which CPUs are for application containers to run workloads. 
 <3> Optional: Specify a node selector to apply the performance profile to specific nodes.
 
+.Additional information
+link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of Guaranteed]
```

Comment:
The heading for this should be "Additional resources" (not "information"), and that anything under it should always be a bullet, even if it's just one. But it looks like you already have one under it in the assembly, so it's appearing back-to-back in the rendering. Can you move this link to there?

Reply:
Updated as suggested.